.github/workflows/main.yml

```
@@ -21,3 +21,20 @@ jobs:
       - uses: actions/checkout@v2
       - run: docker build -t libgusb-${{ matrix.distro }} -f contrib/ci/Dockerfile-${{ matrix.distro }} .
       - run: docker run -t -v `pwd`:/build libgusb-${{ matrix.distro }} ./contrib/ci/build_and_test.sh -Dtests=false
+
+  build-freebsd:
+    runs-on: macos-latest
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Build
+      id: test
+      uses: vmactions/freebsd-vm@v0.1.3
```

You may want to bump the freebsd-vm to FreeBSD 13.0-STABLE.
```suggestion
      uses: vmactions/freebsd-vm@v0.1.4
```